<ui version="4.0" >
 <class>UpdateInstaller</class>
 <widget class="QWidget" name="UpdateInstaller" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update Installer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Installing Updates</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="font" >
      <font>
       <family>Courier</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="lineWrapMode" >
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="cmdOk" >
     <property name="text" >
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdOk</sender>
   <signal>clicked()</signal>
   <receiver>UpdateInstaller</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>204</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel" >
     <x>325</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
